evol/population.py

```diff
@@ -36,11 +37,13 @@ def __init__(self, chromosomes, eval_function, maximize=True, logger=BaseLogger(
         self.intended_size = len(chromosomes) if intended_size is None else intended_size
         self.maximize = maximize
         self.logger = logger
+        self.serializer = SimpleSerializer(target=checkpoint_target) if serializer is None else serializer
```

Comment:
it kind of feels like there are two ways to set a serializer now and technically you can pass two competing settings. you can create a serializer instance and immediately overwrite it. would't it be cleaner to just pass a serializer class just like logging?

Reply:
I do agree that this is a little ugly. On the other hand I don't like the overhead of having to instantiate a new object _just_ to change a setting.

Reply:
I can live with that. If we get many different `Serializers` then I think changing it would be best. I realise that a main difference with the loggers is that there are very different methods of logging things while serialisers will most likely never really change.